curvlinops/inverse.py

```diff
@@ -81,6 +80,81 @@ def _matvec(self, x: ndarray) -> ndarray:
         return result
 
 
+class LSMRInverseLinearOperator(_InverseLinearOperator):
+    """Class for inverse linear operators via LSMR.
+
+    See https://arxiv.org/abs/1006.0758 for details on the LSMR algorithm.
+    """
+
+    def __init__(self, A: LinearOperator):
+        """Store the linear operator whose inverse should be represented.
+
+        Args:
+            A: Linear operator whose inverse is formed.
+        """
+        super().__init__(A.dtype, A.shape)
+        self._A = A
+
+        # LSMR hyperparameters
+        self.set_lsmr_hyperparameters()
+
+    def set_lsmr_hyperparameters(
+        self,
+        damp=0.0,
+        atol=1e-06,
+        btol=1e-06,
+        conlim=100000000.0,
```

Comment:
Can we change this into `1e8` so I don't have to count the zeros every time I want to know the value? 😂